src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -116,6 +123,7 @@ public User getUser(UUID id) {
    */
   public void addUser(User user) {
     users.add(user);
+    if (!user.isAdmin()) activityStore.addActivity(new Activity(user));
```

Comment:
I am a bit confused about the condition here, why don't we add Activity when the user is Admin? Right now the admin is hard coded, but later on, we may add the feature to add more admins (not the hard coded way). But maybe this is good for now? What do you think?

Reply:
Yeah; this was a mistake. I have decided to access a field that allows the code to tell wether an activity is private or public. In this case, I am only displaying public stuff. The other private stuff will be displayed in the appropriate place.